designs/administrator/future-openshift/cluster-settings/index.md

```diff
@@ -0,0 +1,64 @@
+
+## Adding Alerts to the Notification Drawer
+
+![Alerts - Available Updates](img/notifications-updates.png)
+- Today, available updates appear in the message section of the notification drawer. 
+- Three new alerts will be added to inform users if (1) a new patch becomes available, (2) a new minor release becomes available, and/or (3) new channels become available. 
+- All three messages send users to the cluster settings section of the console to take further action.
+
+## Channel Visualization and In Context Help
+
+The new channel visualization provides users with more transparency into recommended upgrade paths, informs users what changes happen in each release, and guides users to the correct channel. 
+
+In addition to the visualization: 
+- Field level help text was added for channels
+- A new manage subscription field was added with a direct link to manage subscriptions in OpenShift Cluster Manager
+
+![Channel help text](img/channel-help.png)
+- Clicking `Channel` surfaces a popover that provides context as to what a channel is along  with a link to documentation to learn more about channels in OpenShift.
+
+![Channel visualization](img/channel-visualization.png)
+- The blue line signifies the current channel you are on. The current version and recommended version (if available) are both marked with blue dots. 
+- The gray line signifies the next available channel.
+- To surface recommended versions in the next available channel, users must switch to the channel by clicking the `Pencil` next to `Channel`. 
+- If no other channels are available the gray line will not be visible. 
+- At this time the visualization only supports surfacing one additional channel - the next available.
+
+![Channel visualization - recommended version popover](img/recommended-channel.png)
+- Clicking on the recommended version, in this case the `4.3.22 dot`, will surface a popover with recommended minor release information and a direct link to release notes.
+
+![Channel visualization - “+ more” popover](img/more-channels.png)
+- Clicking `+ More` will surface a popover that lists out all versions (between the current version and recommended version) in a table with direct links to release notes. 
+
+![Channel visualization - “+ more” modal](img/modal.png)
+- In the case that there are more than 5 versions available (between the current version and recommended version), clicking `+ More` will surface a modal.
+
+## Updating a Channel and Viewing the In Progress Checklist
+
+It was important to provide more transparency to the user when an update is in progress. The mockups below illustrate the steps a user takes to update to a minor release in their current channel, and surfaces the in progress checklist that allows users to see the status of the update in real time.
+
+![Update to minor release](img/update.png)
+- Clicking `Update` launches a modal to confirm the update.
+
+![Update modal](img/update-modal.png)
```

Comment:
Would this placeholder text be the same version description as was shown in the popover?

Reply:
I don't think so. I need to replace with the actual text in the modal today. Thank you for catching that.

Reply:
@itsptk updated the modal content - thanks again for catching that.

Reply:
To that point, should we still have a link out to the docs here for someone who needs more explanation on channels? We have that info in the popover but thinking it makes sense here too possibly if we are worried about discoverability of those underlines.

Reply:
@beanh66 Would something like this work?

<img width="964" alt="Screen Shot 2020-06-30 at 11 09 27 AM" src="https://user-images.githubusercontent.com/62031580/86145206-aa324900-bac4-11ea-922a-6ecc8cd9539e.png">

Reply:
LGTM @megan-hall